src/blocks/opentable/save.js

```diff
@@ -0,0 +1,59 @@
+
+import { transformRIDs } from './opentable';
+
+export default function save( { attributes } ) {
+	const rids = attributes.restaurantIDs;
+	const className = ( attributes.className === undefined ) ? '' : attributes.className.substring( attributes.className.lastIndexOf( '-' ) + 1 );
+	const ridsString = transformRIDs( rids, className );
+	const isMultiple = rids.length > 1;
+
+	const styles = [ 'tall', 'standard', 'button' ];
+	if ( ( attributes.className === undefined || attributes.className === null ) || ( ! styles.includes( className ) ) ) {
```

Comment:
`( attributes.className === undefined || attributes.className === null )`

This could be simplified.
`! attributes?.className`